api/src/test/java/org/openmrs/module/labintegration/api/communication/hl7/messages/ack/AckParserTest.java

```diff
@@ -0,0 +1,55 @@
+package org.openmrs.module.labintegration.api.communication.hl7.messages.ack;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.runners.MockitoJUnitRunner;
+import org.openmrs.module.labintegration.api.communication.hl7.messages.testdata.HL7TestMsgUtil;
+import org.openmrs.module.labintegration.api.hl7.messages.ack.AckParser;
+import org.openmrs.module.labintegration.api.hl7.messages.ack.Acknowledgement;
+import org.openmrs.module.labintegration.api.hl7.messages.ack.InvalidAckException;
+
+import java.io.IOException;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AckParserTest {
+	
+	private static final String ACK_CA_FILE = "ACK_CA.hl7";
```

Comment:
Maybe you could wrap those shortcuts into more easy-to-read form? See other occurences